pkg/collector/corechecks/cluster/orchestrator/collectors/collector.go

```diff
@@ -60,7 +60,8 @@ type CollectorRunConfig struct {
 
 // CollectorRunResult contains information about what the collector has done.
```

Comment:
I would update the comment here to explain how metadata differs from manifests as both a the same interface it is unclear what the content of these vars are

Reply:
Especially how they differ

Reply:
I added more comment on `ProcessResult`
https://github.com/DataDog/datadog-agent/blob/551ba56141bb206e8be1dd6021425594f156abf6/pkg/collector/corechecks/cluster/orchestrator/processors/processor.go#L86-L93

I updated `CollectorRunResult` to contain `ProcessResult` 
https://github.com/DataDog/datadog-agent/blob/551ba56141bb206e8be1dd6021425594f156abf6/pkg/collector/corechecks/cluster/orchestrator/collectors/collector.go#L65-L69

Before it was :
https://github.com/DataDog/datadog-agent/blob/74e56431ccde29ef28075607a15753743f5c7756/pkg/collector/corechecks/cluster/orchestrator/collectors/collector.go#L62-L67